extensions/notebook/package.json

```diff
@@ -628,6 +644,18 @@
         "key": "Ctrl+Shift+T",
         "when": "activeEditor == workbench.editor.notebookEditor"
       },
+      {
+        "command": "notebook.command.undoNotebookView",
+        "key": "ctrl+z",
+        "mac": "cmd+z",
+        "when": "focusedView"
```

Comment:
This is going to show this command for ANY focused view, right? We should only be doing this when the book view is focused.